tasks/util/strict_d3.js

```diff
@@ -0,0 +1,26 @@
+var path = require('path');
+var transformTools = require('browserify-transform-tools');
+var constants = require('./constants');
+
+var pathToStrictD3Module = path.join(
+    constants.pathToImageTest,
+    'strict-d3.js'
+);
+
+/**
+ * Transform `require('d3')` expressions to `require(/path/to/strict-d3.js)`
+ */
+
+module.exports = transformTools.makeRequireTransform('requireTransform',
```

Comment:
@alexcjohnson I hope you don't mind this commit too much. This made incorporating `strict-d3.js` in our jasmine tests _a lot_ easier - which is very useful e.g. to :lock: down https://github.com/plotly/plotly.js/pull/1871/commits/e234827dac252743bb68329487cabaee80dab953

Companion `plotly-mock-viewer` PR: https://github.com/rreusser/plotly-mock-viewer/pull/15